xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -361,6 +361,12 @@ private String extractWikiName(String documentName, XWikiContext context)
         return pageList;
     }
 
+    private QueryParameter getQueryParameter(String like)
+    {
+        String likeString = like.replaceAll("([!])", "!$1");
```

Comment:
Sounds very specific and missing information about the reason of the problem. Why is `!` not working? Because of the FS. Then you'll to reuse a FS path serializer (which already exists).

Reply:
ok so I've checked the jira issue and it's not related to the FS serialization.

We need to verify if the problem is not in the query module since it looks strange that we would need to replace the "!" char here.